### PR TITLE
Add JakobDev.jdMinecraftLauncher

### PR DESCRIPTION
**Pull request by JakobDev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/88878)

### CODE REVIEW

**Comment by JakobDev** [comment]
Is SmartScreen the Problem here?

**Comment by stephengillie** [comment]
> Is SmartScreen the Problem here?

Yes, SmartScreen is prompting during install. Also, the program presents an error when launched: 
image

**Comment by JakobDev** [comment]
> Yes, SmartScreen is prompting during install

What can I do to fix this? My Program is fully OpenSource (including the Installer, which source you can find [here](https://gitlab.com/JakobDev/jdMinecraftLauncher/-/blob/master/deploy/Installer.nsi), and the build script, which you can find [here](https://gitlab.com/JakobDev/jdMinecraftLauncher/-/blob/master/.gitlab-ci.yml)) so I (and everyone who takes a look at the code) can guaranty that this is no malware. And I won't buy a certificate for a very nice OpenSource program that is only interesting for a handful of Users.

> Also, the program presents an error when launched

Thanks for bringing this to my attention. It's working on my system but not one other ones, so I haven't found it by myself. It looks like a Bug in cx-Freeze. I've already opened a [Bug report](https://github.com/marcelotduarte/cx_Freeze/issues/1685).

File: manifests/j/JakobDev/jdMinecraftLauncher/4.1/JakobDev.jdMinecraftLauncher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+PackageIdentifier: JakobDev.jdMinecraftLauncher
+PackageVersion: 4.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Installers:
```
Comment (stephengillie) [inline manifests/j/JakobDev/jdMinecraftLauncher/4.1/JakobDev.jdMinecraftLauncher.installer.yaml:5]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
Installers:
```

**Comment by stephengillie** [comment]
> What can I do to fix this? 
Discussing with coworkers, the SmartScreen prompt isn't a blocker. Once the package's bugs are sorted, and it passes validation, it will be available through Winget. 

> Thanks for bringing this to my attention. It's working on my system but not one other ones, so I haven't found it by myself. It looks like a Bug in cx-Freeze. I've already opened a [Bug report](https://github.com/marcelotduarte/cx_Freeze/issues/1685).

Happy to help. In a VM, the application needed the Visual C++ dependencies also. These would be the next hurdle in the validation process after install.

**Comment by JakobDev** [comment]
> Discussing with teammates, the SmartScreen prompt isn't a blocker. Once the package's bugs are sorted, and it passes validation, it will be available through Winget.

Good to hear.

> Happy to help.

Thanks

> In a VM, the application needed the Visual C++ dependencies also. These would be the next hurdle in the validation process after install.

This should be no problem. [cx-Freeze provides a option](https://cx-freeze.readthedocs.io/en/latest/faq.html#microsoft-visual-c-redistributable-package) to include it. I can activate it and do a rebuild.

So the only thing that blocks it, is that Bug from cx-Freeze, that launching failed. I sadly didn't get a response to my bug report yet.

File: manifests/j/JakobDev/jdMinecraftLauncher/4.1/JakobDev.jdMinecraftLauncher.installer.yaml (hunk 1 above)
Comment (JakobDev) [reply manifests/j/JakobDev/jdMinecraftLauncher/4.1/JakobDev.jdMinecraftLauncher.installer.yaml:5]: I will include this in the Build as explained below

**Review by ItzLevvie — DISMISSED** [review]

**Comment by stephengillie** [comment]
Please feel free to mark as Ready for review once the publisher has released the bug fix.

**Comment by JakobDev** [comment]
I'm still working on this. Communication with the Maintainer of cx-freeze is a little bit slow.

**Comment by JakobDev** [comment]
I had now build a new Version using PyInstaller

**Review by stephengillie — APPROVED** [review]

**Comment by JakobDev** [comment]
@stephengillie I thinks this needs some human intervention

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**